app/controller/sdl/MediaController.js

```diff
@@ -77,26 +77,42 @@ SDL.SDLMediaController = Em.Object.create(
       // set active model
       SDL.SDLController.set('model', applicationModel);
 
-      if (SDL.SDLController.model.appType) {
-        for (var i = 0; i < SDL.SDLController.model.appType.length; i++) {
-          if (SDL.SDLController.model.appType[i] == 'NAVIGATION' ||
-              SDL.SDLController.model.appType[i] == 'PROJECTION') {
-            this.model.currentAudioModel.turnOnSDL();
-            SDL.BaseNavigationView.update();
-            SDL.States.goToStates('navigationApp.baseNavigation');
-            return;
-          }
-          if (SDL.SDLController.model.appType[i] == 'TESTING') {
-            // TODO(WEBENGINE)
-            SDL.InfoController.turnOnSDL();
-            SDL.States.goToStates('webViewApp');
-            return;
+      let get_template_from_app_type = function() {
+        if (SDL.SDLController.model.appType) {
+          for (var i = 0; i < SDL.SDLController.model.appType.length; i++) {
+            if (SDL.SDLController.model.appType[i] == 'NAVIGATION' ||
+                SDL.SDLController.model.appType[i] == 'PROJECTION') {
+                return 'NAV_FULLSCREEN_MAP';
+              }
+              if (SDL.SDLController.model.appType[i] == 'TESTING') {
+                // TODO(WEBENGINE)
```

Comment:
Is it necessary here?

Reply:
@YarikMamykin yes, it is. This is a labels in a code where `TESTING` should be replaced with `WEB_VIEW` in the production-ready feature. This will be done in a separate PR once #16 and #17 will be merged into the feature branch